lambda/custom/state_handlers.js

```diff
@@ -115,6 +126,22 @@ var state_handlers = {
     'AMAZON.PauseIntent': function() {
       console.log('PLAY_MODE:AMAZON.PauseIntent');
       audio_controller.stop.call(this);
+      this.emit('StoppedArticle');
+    },
+    StoppedArticle: function() {
+      console.log('PLAY_MODE:StoppedArticle');
+      if (this.attributes['full']) {
+        scout_agent
+          .updateArticleStatus(
+            this.attributes['userId'],
+            this.attributes['articleId'],
+            this.attributes['offsetInMilliseconds']
+          )
+          .catch(function(err) {
+            console.log('Error during offset update.');
```

Comment:
Another catch which is hiding error details -- take a look at this and the other ones in the file. If this is an unexpected error, try to provide us some more details when we are troubleshooting this via log files somewhere down the line.